primitives/std.lib

```diff
@@ -822,11 +844,12 @@ primitive fixed_p_std_const[width, int_width, fract_width, value1, value2] () ->
         module fixed_p_std_const
             #(parameter width=32,
             parameter int_width = 8,
-            parameter fract_width= 24,
+            parameter fract_width = 24,
             parameter value1 = 0,
             parameter value2 = 0)
 
         (output logic [width-1:0] out);
+        /* verilator lint_off WIDTHCONCAT */
```

Comment:
What is this for?

Reply:
Ah, thanks for catching this. That was a "simple fix" to some error I was getting when using the `std_exp`primitive. I never really looked more into it, mostly because I forgot about it. It shouldn't be necessary when I instead use invoke with a FuTIL component.